templates/hooks/certificate_authority/sdk_update_pre_build_request.go.tpl

```diff
@@ -0,0 +1,9 @@
+	if delta.DifferentAt("Spec.Tags") {
+		err := rm.syncTags(ctx, desired, latest)
+		if err != nil {
+			return nil, err
+		}
+		if !delta.DifferentExcept("Spec.Tags") {
+			return desired, nil
+		}
```

Comment:
This block should be outside the if it self. Check https://github.com/aws-controllers-k8s/acm-controller/blob/38ecf7c3346c9066e5dc52d4406f9bfdbed1d46c/templates/hooks/certificate/sdk_update_pre_build_request.go.tpl#L11-L14